### ISSUE TITLE
After version 5.3.2 can't open templates that have pivot tables

### ISSUE THREAD

**Issue by charneca** [EPPlusSoftware/EPPlus#261]
When I open an Excel that have one or more pivot tables, after saving im getting this error:

 excelFile.SaveAs(finalFile);

'An item with the same key has already been added. Key: '

before version 5.4.0 it is ok.

**Comment by charneca** [comment]
I also have another issue after version 5.0.4

After changing an excel that have a pivot table with conditional formating.

Excel complains that found a problem then gives this message:
> Repaired Records: Conditional formatting from /xl/pivotTables/pivotTable1.xml part (PivotTable view)

**Comment by JanKallman** [comment]
Hello, 
Issue 1 is solved and will be released with the next version. Your second issue i'm not sure. Can you attach your workbook and some code to reproduce?

**Comment by charneca** [comment]
Sure for the test I have used .net5 and epplus 5.3.2
Here is the sample excel and code:

[test.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5708157/test.xlsx)

```
FileInfo template = new FileInfo(workingDirectory + @"test.xlsx");
FileInfo finalFile = new(string.Format(workingDirectory + @"test{0}.xlsx", DateTime.Now.Ticks));
using var excelFile = new ExcelPackage(template) ;
var ws = excelFile.Workbook.Worksheets["data"];
ws.Cells["A1"].Value = "test";
excelFile.SaveAs(finalFile);
```

**Comment by JanKallman** [comment]
If the conditional formatting accesses pivot tables and had multiple cfRule elements accessing the same range and the range contained multiple comma separated references, this issue occured. Fixed applied

**Comment by charneca** [comment]
thank you, keep up the great work!

**Comment by JanKallman** [comment]
Fixed in version 5.5.1
